network/messages_test.cc

```diff
@@ -19,173 +19,176 @@ limitations under the License.
 #include <iostream>
 #include "messages.h"
 
+namespace Network
+{
 namespace
 {
 
 TEST(MessagesTest, WriteAndReadUint32)
 {
-    Network::Buffer buf;
-    uint32_t        write_value = 123456703;
-    Network::WriteUint32ToBuffer(write_value, buf);
+    Buffer         buf;
+    const uint32_t write_value_1 = 123456703;
+    WriteUint32ToBuffer(write_value_1, buf);
     size_t offset = 0;
-    auto   read_value = Network::ReadUint32FromBuffer(buf, offset);
-    ASSERT_TRUE(read_value.ok());
-    ASSERT_EQ(write_value, *read_value);
+    auto   read_value_1 = ReadUint32FromBuffer(buf, offset);
+    ASSERT_TRUE(read_value_1.ok());
+    ASSERT_EQ(write_value_1, *read_value_1);
```

Comment:
Consider using status_matchers like #285.

```suggestion
    ASSERT_THAT(Network::ReadUint32FromBuffer(buf, offset), IsOkAndHolds(write_value_1));
```

It's more succinct, readable, and produces better output. Before

```
[ RUN      ] MessagesTest.WriteAndReadUint32
/usr/local/google/home/hitchens/git/dive/network/messages_test.cc:35: Failure
Value of: read_value.ok()
  Actual: false
Expected: true

[  FAILED  ] MessagesTest.WriteAndReadUint32 (0 ms)
```

After:

```
[ RUN      ] MessagesTest.WriteAndReadUint32
/usr/local/google/home/hitchens/git/dive/network/messages_test.cc:34: Failure
Value of: Network::ReadUint32FromBuffer(buf, offset)
Expected: is OK and has a value that is equal to 123456703
  Actual: ( INVALID_ARGUMENT: Buffer too small to read a uint32_t. ), which has status INVALID_ARGUMENT: Buffer too small to read a uint32_t.

[  FAILED  ] MessagesTest.WriteAndReadUint32 (0 ms)
```